feconf.py

```diff
@@ -445,10 +453,13 @@ def get_empty_ratings():
 FEEDBACK_THREADLIST_URL_PREFIX = '/threadlisthandler'
 FEEDBACK_THREAD_VIEW_EVENT_URL = '/feedbackhandler/thread_view_event'
 FLAG_EXPLORATION_URL_PREFIX = '/flagexplorationhandler'
+LIBRARY_GROUP_INDEX_URL = '/librarygroupindexhandler'
```

Comment:
and use /librarygrouphandler. The 'index' actually refers to the 'index page', but this isn't an index page.